addons/bus/static/tests/helpers/websocket_event_deferred.js

```diff
@@ -123,11 +123,14 @@ function _waitNotification(notification) {
             if (notification.type !== type) {
                 continue;
             }
-            if (JSON.stringify(notification.payload) === JSON.stringify(payload)) {
+            if (
+                payload === undefined ||
```

Comment:
So this is losing the ability to check/wait for specifically empty payload?

Reply:
Quite the opposite actually. Now, we can omit the payload if we don't care about it or if it is supposed to be empty.